### PR TITLE
Prepare for release 2.1.0

### LINKED ISSUE #1738: Plan for a new release?
Hi,

I see that the latest release for resque was over 2 years ago. There are some features in code (like worker_exit hook), that is not available in the 2.0.0 build. Is there a plan to go for another release?

Thanks,
Aman

### PR DESCRIPTION
**Pull request by jrochkind**
Closes #1738

### CODE REVIEW

**Comment by jrochkind** [comment]
Recommend maintainers insist on an edit to HISTORY.md before merging PR's in future, so it's done as we go.

File: CONTRIBUTING.md
Comment (iloveitaly) [inline CONTRIBUTING.md:15]: 👍

File: HISTORY.md
Comment (iloveitaly) [inline HISTORY.md:33]: nit: space between issue number

**Review by iloveitaly — CHANGES_REQUESTED** [review]
Thank you! Two minor nits and then we should be good to go.

File: CONTRIBUTING.md
Comment (jrochkind) [reply CONTRIBUTING.md:23]: I didn't like leaving it suggesting that a 2015 post by klabnik (who isn't involved anymore?) describes the current direction and state of resque. We know it does *not*, and I think we need to say so instead of let it imply that.  And also I like acknowledging that the current state of and direction of resque is in fact not very transparent to users, and we know that could be improved. 

Maybe there's a different way to say that then the "TODO"?  I worry just eliminating the TODO without something else leaves it seeming to tell readers that the 2015 doc by klabnick actually does describe resque current state and direction. 

But your call!

Comment (jrochkind) [reply CONTRIBUTING.md:23]: Like honestly, for me part of it is when I see the statement "see teh state and direction of resque here" and it's from 2015 and no longer up to date -- I assume the project is abandoned!  When I see someone made a commit saying "Yeah, we know we need to update this" then I think "oh, someone is home, the lights are on!"

Obviously, actually updating it would be better. You may be the only person with the context to do so!

Comment (chrisccerami) [reply CONTRIBUTING.md:23]: Honestly, I would just remove those few lines entirely that link to Steve's post.

Comment (jrochkind) [reply CONTRIBUTING.md:23]: OK, your call, removing it! 

Not necessary to resolve in this PR, but I do think it would be good to put somethign in the README about current status and direction of resque, as this line used to do in 2015.  As a resque user, it was difficult to figure out if it was currently maintained etc, and I spent time trying to figure out what was going on that could be saved with a little explanation!

Comment (chrisccerami) [reply CONTRIBUTING.md]: I think part of the issue is that a lot of the maintainers from a few years ago have moved on to working on other things, and we haven't had a lot of newer maintainers join. I don't know that there really is a direction for it at the moment. I'd be happy to bring some new maintainers on board who are eager to work on Resque.

**Comment by jrochkind** [comment]
> I think part of the issue is that a lot of the maintainers from a few
years ago have moved on to working on other things, and we haven't had a
lot of newer maintainers join. I don't know that there really is a
direction for it at the moment. I'd be happy to bring some new maintainers
on board who are eager to work on Resque.

Totally! Exactly what you described there is kind of hard for a resque user
to find out, but is very useful for a user or potential user to know. I am
just suggesting some text explaining this be added somewhere, such as the
README.  Including maybe a couple other sentances? How many
active/authorized maintainer/committers are there at present, just one, who
are they? What is their/your own time/energy for resque at present
(caretaker/trying-to-keep-the-lights on only? More? Realistically, planning
on responding to PR's or not? Etc.).   I think it's super helpful to users
and the community to just be transparent about all of that.  Something
useful can be done in just a few sentances.

And @iloveitaly is an obvious candidate for "new maintainers who are eager
to work on Resque". :)

**Comment by chrisccerami** [comment]
> I think part of the issue is that a lot of the maintainers from a few
> years ago have moved on to working on other things, and we haven't had a lot of newer maintainers join. I don't know that there really is a direction for it at the moment. I'd be happy to bring some new maintainers on board who are eager to work on Resque. Totally! Exactly what you described there is kind of hard for a resque user to find out, but is very useful for a user or potential user to know. I am just suggesting some text explaining this be added somewhere, such as the README. Including maybe a couple other sentances? How many active/authorized maintainer/committers are there at present, just one, who are they? What is their/your own time/energy for resque at present (caretaker/trying-to-keep-the-lights on only? More? Realistically, planning on responding to PR's or not? Etc.). I think it's super helpful to users and the community to just be transparent about all of that. Something useful can be done in just a few sentances. And @iloveitaly is an obvious candidate for "new maintainers who are eager to work on Resque". :)

Absolutely. I'll open an issue where people can discuss this further (future direction, volunteer to help out, etc) and link it in the README with some details. In the meantime, I'll try to figure out how to get @iloveitaly set up.

**Comment by chrisccerami** [comment]
I've created [this PR](https://github.com/resque/resque/pull/1760) to update the README and [this issue](https://github.com/resque/resque/issues/1759) for discussion.

**Comment by jrochkind** [comment]
Sweet thanks! As far as this PR, I did remove the reference to the 2015
post, and I think it's ready, for then a subsequent resque 2.1.0 release?
(woah)

**Comment by iloveitaly** [comment]
@jrochkind thanks for this! Merging and cutting a new release.

**Pull request merged**
